File: In_C_Programming/q18.c
#include <stdio.h>

void get_Element(int arr[], int ln)
{

    printf("Enter Array Elements: ");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }
}

int get_Max(int arr[], int ln)
{
    int max = arr[0];

    for (int i = 1; i < ln; i++)
    {
        if (arr[i] > max)
        {
            max = arr[i];
        }
    }

    return max;
}

int get_Min(int arr[], int ln)
{
    int min = arr[0];

    for (int i = 1; i < ln; i++)
    {
        if (arr[i] < min)
        {
            min = arr[i];
        }
    }

    return min;
}

int main()
{

    int ln;

    printf("Enter The Array Length: ");
    scanf("%d", &ln);

    int arr[ln];

    get_Element(arr, ln);

    int big = get_Max(arr, ln);
    int small = get_Min(arr, ln);

    printf("The Difference is: %d \n", big - small);

    return 0;
}

File: In_Java_Programming/q18.java
import java.util.Scanner;
import java.util.Vector;

public class q18 {

    public static int get_Maximum(Vector<Integer> vec, int ln) {

        int max = Integer.MIN_VALUE;

        for (Integer i : vec) {

            if (i > max) {
                max = i;
            }
        }
        return max;
    }

    public static int get_Minimum(Vector<Integer> vec, int ln) {

        int min = Integer.MAX_VALUE;

        for (Integer i : vec) {

            if (i < min) {
                min = i;
            }
        }
        return min;
    }

    public static void get_Element(Vector<Integer> vec, int ln, Scanner sc) {
        System.out.println("Enter The Array Elements: ");

        for (int i = 0; i < ln; i++) {

            int num = sc.nextInt();
            vec.addLast(num);
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        Vector<Integer> vec = new Vector<Integer>();

        get_Element(vec, ln, sc);

        int max = get_Maximum(vec, ln);
        int min = get_Minimum(vec, ln);

        System.out.println("The Difference is: " + (max - min));

        sc.close();
    }
}
